File: src/main/java/com/estonianport/agendaza/AgendazaApplication.java
package com.estonianport.agendaza;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

import com.ulisesbocchio.jasyptspringboot.annotation.EnableEncryptableProperties;

@SpringBootApplication
@EnableEncryptableProperties
public class AgendazaApplication {

	public static void main(String[] args) {
		SpringApplication.run(AgendazaApplication.class, args);
	}

}

File: src/main/kotlin/com/estonianport/agendaza/AgendazaApplication.kt
package com.estonianport.agendaza

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer

@SpringBootApplication
class AgendazaApplication : SpringBootServletInitializer()

     fun main(args: Array<String>) {
        runApplication<AgendazaApplication>(*args)
}
